sql-statements/sql-statement-create-table.md

```diff
@@ -215,10 +231,14 @@ The `table_option` currently only supports `AUTO_INCREMENT`, `SHARD_ROW_ID_BITS`
 | `AUTO_INCREMENT` | The initial value of the increment field | `AUTO_INCREMENT` = 5 |
 |`SHARD_ROW_ID_BITS`| To set the number of bits for the implicit `_tidb_rowid` shards |`SHARD_ROW_ID_BITS` = 4|
 |`PRE_SPLIT_REGIONS`| To pre-split `2^(PRE_SPLIT_REGIONS)` Regions when creating a table |`PRE_SPLIT_REGIONS` = 4|
+|`AUTO_ID_CACHE`| To set the auto ID cache size in a TiDB instance. By default, TiDB automatically resize it according to allocation speed |`AUTO_ID_CACHE` = 200|
```

Comment:
```suggestion
|`AUTO_ID_CACHE`| To set the `AUTO_ID` cache size in a TiDB instance. By default, TiDB automatically changes this size according to the allocation speed of `AUTO_ID` |`AUTO_ID_CACHE` = 200|
```

Reply:
`AUTO_ID` is not a keyword. "auto ID" means "ID allocated automatically", including `auto_increment` ID and `auto_random` ID.